common-practices-tools/security/encryption.md

```diff
@@ -1,186 +1,50 @@
-
-[Mailvelope](https://www.mailvelope.com/) integrates GPG with your Gmail using a Chrome or Firefox extension.
-
-### GPG Suite (for Mac OSX Mail App)
-
-See [GPGTools](https://gpgtools.org/) _(not yet fully integrated with Sierra)_
-
-### Enigmail (Mozilla Thunderbird)
-
-[Enigmail](https://www.enigmail.net/index.php/en/) works with Mozilla Thunderbird and GPG to deliver a seamless encrypted email experience.
-
-### More Email References
-
--   [Email Self-Defense](https://emailselfdefense.fsf.org/en/) (Free Software Foundation)
--   [The Best Free Ways to Send Encrypted Email and Secure Messages](http://www.howtogeek.com/135638/the-best-free-ways-to-send-encrypted-email-and-secure-messages/) (HowToGeek)
--   [Why No One Uses Encrypted Email Messages](http://www.howtogeek.com/187961/why-no-one-uses-encrypted-email-messages/) (HowToGeek)
--   [Why You Should Encrypt Your Email](https://www.lifewire.com/you-should-encrypt-your-email-2486679) (LifeWire)
-
-#### ProtonMail
-
-[ProtonMail](https://protonmail.com/) is a free and secure email service that provides arguably better security than GPG encrypted email sent via (say) Gmail as it not only encrypts but the service does not harvest/index/resell metadata, such as who is emailing whom and when. For the best results, both sides must be using ProtonMail.
+[ProtonMail](https://protonmail.com/) is a free and secure email service that provides arguably better security than GPG encrypted email sent via (say) Gmail as it not only encrypts but the service does not harvest/index/resell metadata, such as who is emailing whom and when. To obtain true end-to-end encryption, both sender and receiver must be using ProtonMail.
 
 ## Private Browsing
 
-Using [Tor](https://www.torproject.org/), the [CivicActions VPN](https://git.civicactions.net/devops/internal-it-wireguard-vpn/tree/master) (for CivicActions business) or a paid VPN (such as [PIA](https://www.privateinternetaccess.com/) and there are others; be sure to choose an exit node that is not in the U.S.) is the best route, but the two simple and free services below will have little impact on your browsing experience and can go a long way to minimizing the data others collect about you:
-
-We recommend using both of these browser add-ons.
+We recommend using both of these browser add-ons that enhance privacy without encryption:
 
 -   [Privacy Badger](https://www.eff.org/privacybadger) blocks most ads and cookie collectors. Open source and free.
 -   [uBlock Origin](https://ublockorigin.com) is an ad content blocker. Open source and free.
 
-For search, we recommend changing your browser "default search" to [DuckDuckGo](https://duckduckgo.com/) or [Ecosia](https://www.ecosia.org/) as they protects privacy and avoids the filter bubble.
+For search, we recommend changin
+g your browser "default search" to [DuckDuckGo](https://duckduckgo.com/) or [Ecosia](https://www.ecosia.org/) as they protects privacy and avoid the filter bubble.
```

Comment:
Small typo here, the `g` is on a new line which is translating into a space on the site https://civicactions-handbook--1055.org.readthedocs.build/en/1055/common-practices-tools/security/encryption/#private-browsing.